src/_component/skills/skills.scss

```diff
@@ -0,0 +1,46 @@
+.skills__list {
+  border-radius: 6px;
+  border: 2px solid #707070;
+  list-style-type: none;
+}
+
+.skills__item {
+  display: flex;
+  border: 2px solid #707070;
+
+  &:first-child .skills__icon {
+    color:#d52027;
+  }
+
+  &:nth-child(2) .skills__icon {
+    color: #0caf0c;
+  }
+
+  &:last-child .skills__icon {
+    color: #1875f0;
+  }
+
+}
+
+.skills__count {
+  color: #666666;
+  font-size: 16px;
+}
+
+.skills__count,
+.skills__icon {
+  padding: 10px 12px 8px 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.skills__label {
+  line-height: 48px;
+  color: #666666;
+}
+
+.skills__label {
+  flex-grow: 1;
+}
+
```

Comment:
Why two times? Refactor to only one selector declaration.

Reply:
Done! :)